src/modules/ipc.cpp

```diff
@@ -59,11 +60,20 @@ namespace modules {
     for (auto& hook : m_hooks) {
       hook->command = pid_token(hook->command);
     }
-    m_formatter->add(DEFAULT_FORMAT, TAG_OUTPUT, {TAG_OUTPUT});
+    m_formatter->add(DEFAULT_FORMAT, TAG_LABEL, {TAG_LABEL});
```

Comment:
This breaks any module that manually defined a `format`. For example the config

```dosini
[module/ipc]
type = custom/ipc
hook-0 = echo 0
hook-1 = echo 1
hook-2 = echo 2

format = |<output>|

initial = 1
```

This module showed the hook output in between pipe characters, but in this PR, it only displays `||`.

We still need to support eh `<output>` tag (just as in the text module) and emit a deprecation warning if it is used.